### DISCUSSION TITLE
The character 0x0x93 0x37 encoded in “GBK” has no corresponding value in the code “UTF8”

### DISCUSSION THREAD

**Discussion by k892050089** [MIT-LCP/mimic-code#1450] [MIMIC-IV]
When I was building concepts for MIMIC IV, an error occurred when creating the table "meld" and "sirs" by running the files "meld.sql" in the "organfailure" dictionary and "sirs.sql" in the "score" dictionary. The error occurred when SQL shell reading character " ; " in the last line in each .sql file. As a consquence, the corresponding tables were not installed. How should I solve this problem?

**Comment by k892050089** [comment]
It haved been solved by running the code " \encoding 'UTF8' " in SQL shell before running these two files.

**Reply by zheng523** [reply]
I'm looking for solution for the same problem. Thank you for the feedback.
